osism/commands/manage.py

```diff
@@ -251,30 +257,26 @@ def get_parser(self, prog_name):
 
     def take_action(self, parsed_args):
         wait = not parsed_args.no_wait
-        cloud = parsed_args.cloud
-        dry_run = parsed_args.dry_run
-        filter = parsed_args.filter
-        hide = parsed_args.hide
-        latest = parsed_args.latest
-        images = parsed_args.images
 
         arguments = []
-        if cloud:
+        if parsed_args.cloud:
             arguments.append("--cloud")
-            arguments.append(cloud)
-        if filter:
+            arguments.append(parsed_args.cloud)
+        if parsed_args.filter:
             arguments.append("--filter")
-            arguments.append(filter)
-        if dry_run:
+            arguments.append(parsed_args.filter)
+        if parsed_args.delete:
+            arguments.append("--delete")
+        if parsed_args.dry_run:
             arguments.append("--dry-run")
-        if latest:
+        if parsed_args.latest:
             arguments.append("--latest")
-        if hide:
+        if parsed_args.hide:
             arguments.append("--hide")
 
-        if images:
+        if parsed_args.images:
             arguments.append("--images")
```

Comment:
nit: this is duplicated in both branches, you could move it in front of the conditional